### PR TITLE
MW-248: Style Nav Bar

### PR DESCRIPTION
**Pull request by frlzjosh**
hope it looks good...

### CODE REVIEW

File: dist/css/meta-ui.css [hunk 1]
```diff
@@ -1,5 +1,13 @@
 @import url(https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800);@charset "UTF-8";
 
+.main--fullpage {
+  min-height: calc(100vh - 80px);
+}
+
+.fa {
```
Comment (jasmine1569) [inline dist/css/meta-ui.css]: What is this? If you are trying to change the color of the icons you can just use one of the type color classes in type.scss. Treat them if they were fonts.

File: docs/navbar.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+            </div>
+            <div class="col-md-9">
+                <div class="navbar--top  clearfix">
+                    <img class="margin-left navbar__icon-style1 pull-left" src="../img/logo/metalab-icon.png"
+                         alt="metalab-icon">
+                    <a class="nav-top pull-right"><i class="fa fa-bars fa-lg nav-top--bars icon--padding"
+                                                     aria-hidden="true"></i></a>
+                </div>
+                <div class="navbar__1--documentation">
+                    <h3>DOCUMENTATION FOR TOP BAR</h3>
```
Comment (jasmine1569) [inline docs/navbar.html]: you need to add the class h5 into this so it looks like the other pages

File: src/sass/meta-ui.scss [hunk 3]
```diff
@@ -14,10 +16,19 @@
...
 // COMPONENTS
 @import "footer";
 @import "sidebar";
 @import "banner";
 @import "buttons";
 
+@import "icon";
+
+//NAV BAR
+@import "navbar";
```
Comment (jasmine1569) [inline src/sass/meta-ui.scss]: this belongs underneath components where buttons and the others are at

File: src/sass/meta-ui.scss [hunk 4]
```diff
@@ -14,10 +16,19 @@
...
+
+//clearfix
+@import "clearfix";
```
Comment (jasmine1569) [inline src/sass/meta-ui.scss]: clearfix is a mixin, so it should be in a mixin folder and then clearfix.scss

File: docs/navbar.html (hunk 2 above)
Comment (frlzjosh) [reply docs/navbar.html]: okay i did it

File: src/sass/meta-ui.scss (hunk 3 above)
Comment (frlzjosh) [reply src/sass/meta-ui.scss]: aight

File: src/sass/meta-ui.scss (hunk 4 above)
Comment (frlzjosh) [reply src/sass/meta-ui.scss]: Brian made the clearfix.scss and the branch that he made it in has not been merged into dev yet.

Comment (frlzjosh) [reply src/sass/meta-ui.scss]: so i just made a .scss file and imported it temporarily

Comment (frlzjosh) [reply src/sass/meta-ui.scss]: nvm i did it

**Comment by frlzjosh** [comment]
i fixed merged conflicts

**Pull request merged**